liiweb/templates/liiweb/legislation_list.html

```diff
@@ -60,7 +68,7 @@ <h1 class="my-4">{% trans 'Legislation' %}</h1>
         <div class="alert alert-danger">{% trans 'You are viewing repealed legislation which is no longer in force.' %}</div>
       {% endif %}
       <div data-vue-component="LegislationTable"
-           data-initial-sort="{{ legislation_list_sort }}"
+           data-initial-sort="{% block legislation-list-sort %}{{ legislation_list_sort }}{% endblock %}"
```

Comment:
you shouldn't need this, simply override the variable in the view, there's provision for that

Reply:
ah right, I've set it in KL